src/components/AutosuggestDeprecated/index.js

```diff
@@ -1,2 +0,0 @@
-export { AutosuggestDeprecated } from './AutosuggestDeprecated';
```

Comment:
Did you mark PR that it has breaking changes?

Reply:
:+1: